article/utils/rename_dictionary_keys.py

```diff
@@ -0,0 +1,115 @@
+correspondencia_journal = {
+    "v5": "type_of_literature",
+    "v6": "treatment_level",
+    "v10": "center_code",
+    "v20": "national_code",
+    "v30": "identification_number",
+    "v35": "issn_type",
+    "v37": "secs_code",
+    "v50": "publication_status",
+    "v51": "journal_status_history_in_this_collection",
+    "v62": "copyright_holder",
+    "v63": "address",
+    "v64": "electronic_address",
+    "v66": "ftp",
+    "v67": "user_subscription",
+    "v68": "acronym",
+    "v69": "url_of_the_journal",
+    "v85": "controled_vocabulary",
+    "v90": "notes",
+    "v100": "publication_title",
+    "v110": "subtitle",
+    "v117": "standard",
+    "v130": "section",
+    "v140": "sponsor",
+    "v150": "short_title",
+    "v151": "iso_short_title",
+    "v230": "parallel_titles",
+    "v240": "other_titles",
+    "v301": "initial_date",
+    "v302": "initial_volume",
+    "v303": "initial_number",
+    "v304": "terminate_date",
+    "v305": "final_volume",
+    "v306": "final_number",
+    "v310": "publisher_country",
+    "v320": "publisher_state",
+    "v330": "level_of_publication",
+    "v340": "alphabet",
+    "v350": "text_idiom",
+    "v360": "abstract_language",
+    "v380": "frequency",
+    "v400": "issn_id",
+    "v420": "medline_code",
+    "v421": "medline_short_title",
+    "v430": "classification",
+    "v435": "issn",
+    "v440": "subject_descriptors",
+    "v441": "study_area",
+    "v450": "indexing_coverage",
+    "v480": "publisher",
+    "v490": "publisher_city",
+    "v540": "identifying_regular_issues",
+    "v550": "has_supplement",
+    "v541": "license_of_use",
+    "v560": "is_supplement",
+    "v610": "old_title",
+    "v690": "url_of_the_main_collection",
+    "v691": "scielo_net",
+    "v692": "url_of_submission_online",
+    "v699": "publishing_model",
+    "v710": "new_title",
+    "v851": "web_of_knowledge_databases",
+    "v852": "web_of_knowledge_databases",
+    "v853": "web_of_knowledge_databases",
```

Comment:
@samuelveigarangel corresp.values() não pode ter valores repetidos. Use os nomes das bases de dados.

Reply:
Na documentacao, ta informando que esses  fields, possuem o mesmo valor.
`Web of Knowledge Databases
Field: 851, 852, 853`